pytorch_lightning/trainer/data_loading.py

```diff
@@ -374,7 +374,7 @@ def determine_data_use_amount(self, overfit_batches: float) -> None:
         if overfit_batches > 0:
             if isinstance(overfit_batches, float) and overfit_batches > 1:
```

Comment:
```suggestion
            if isinstance(overfit_batches, float) and overfit_batches >= 1:
```

Reply:
I think if overfit_batches = 1.0 then all is good, no error needs to be raised. Only when it is > 1.0 then we need to raise because now we can't interpret it as a percentage anymore. 
But it is still a good comment, because now I see we should update the error message below to say 
`0.0 < x <= 1.0` instead of `0.0 < x < 1.0`. 
Does that make sense?

Reply:
if overfit_batches == 1.0 it's fine... this is 100%. But anything above that needs to be an int